build/build.proj

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <Configuration>Debug</Configuration>
+    <Platform>&quot;Any CPU&quot;</Platform>
```

Comment:
I will switch the Platform tag to be Mixed Platforms instead of Any CPU. However, I am fairly certain that it would not work unless I had added quotes around the "Mixed Platforms". I will take a look at this again though.